src/Xpress.jl

```diff
@@ -92,7 +92,15 @@ module Xpress
 
         # some lics require special check
         # -------------------------------
-        userlic()
+        xpauth_path = ""
+        if is_unix()
+          if haskey(ENV, "XPRESSDIR")
+            xpauth_path = joinpath(ENV["XPRESSDIR"], "bin")
+          else
+            xpauth_path = joinpath(dirname(dirname(xprs)), "bin")
+          end
+        end
+        userlic(xpauth_path=xpauth_path)

```

Comment:
what do you think: https://gist.github.com/joaquimg/9fb3cea183c16254bb5b072cf1a3ddaa ?